package.json

```diff
@@ -35,7 +35,9 @@
       ">0.2%",
       "not dead",
       "not op_mini all",
-      "not ios_saf < 13"
+      "not ios_saf < 13",
+      "not android 4.4.3-4.4.4",
+      "not ie 11"
```

Comment:
could we say not ie altogether :laughing:

Reply:
all other versions are covered by "not dead" entry, but sure we can have something like 
```
      "not android <= 4.4.4",
      "not ie <= 11"
```

Reply:
o lol dead